package/windows/bootstrap.ps1

```diff
@@ -89,7 +89,6 @@ try
 {
     Copy-Item -Force -Destination "$($hostPrefixPath)etc\rancher" -Path @(
         "c:\etc\rancher\utils.psm1"
-        "c:\etc\rancher\cleanup.ps1"
```

Comment:
Should we still leave this script in in case someone wants to run manual cleanup?

Reply:
it doesnt really work and the new one can be pulled from the container with 1 command and will be documented but the gist is `docker run rancher/rancher-agent -- agent clean scripts > cleanup.ps1` will get you what you need